_scalingo_api/deployments.md

```diff
@@ -52,6 +53,7 @@ Example object:
   "git_ref": "abcdef1234567890",
   "id": "123e4567-e89b-12d3-a456-426655440000",
   "image_size": 104857600,
+  "stack_base_image": "scalingo/ubuntu-20-4",
```

Comment:
Shouldn't this be:
```suggestion
  "stack_base_image": "scalingo/ubuntu-20:v4",
```

Reply:
The string format is arbitrary, so it could be both (test env use the the first for ex), even if indeed, for production envs it will be likely closer to the second one

Reply:
We generally use production like examples.